higher-order-functions/min-max-sorted.md

```diff
@@ -0,0 +1,342 @@
+# Passing Functions as Arguments: Min, Max, and Sort
+
+## Learning Goals
+
+* Demonstrate how to pass a function as an argument
+* Describe how to use the key parameter in the `sorted` function to sort a list of dictionaries by a specific key
+* Describe how to use the key parameter in the `min` function to find the minimum in a list of dictionaries by a specific key
+
+## Introduction
+
+Imagine we have a list of movie titles we'd like to sort alphabetically. Python provides a couple convenient functions for this task. The [`sort`](https://docs.python.org/3/howto/sorting.html#sorting-basics) method can be called on a list and sorts the list in place. The `sorted` function takes a list as a parameter and returns the sorted list.  
```

Comment:
"The `sorted` function takes a list as a parameter and returns the sorted list."
- It might be worth being specific that `sorted` accepts any iterable data type, since that is significantly different from `sort` as a list instance method.